Library/Formula/ltsa.rb

```diff
@@ -0,0 +1,19 @@
+class Ltsa < Formula
+  desc "LTS Analyser"
+  homepage "http://www.doc.ic.ac.uk/~jnm/book/ltsa/LTSA_applet.html"
+  url "http://www.doc.ic.ac.uk/~jnm/book/ltsa/ltsatool.zip"
```

Comment:
If an author updates their package and our downloads break, that's a frustrating experience for users and for us, since we have to find out why it changed and whether it was intentional. Unversioned downloads are also a red flag that a project isn't very mature.